docs/api/alerting/update.asciidoc

```diff
@@ -0,0 +1,139 @@
+[[rules-update-api]]
+=== Update rule API
+++++
+<titleabbrev>Update rule</titleabbrev>
+++++
+
+Update the attributes for an existing rule.
+
+[[rules-update-api-request]]
+==== Request
+
+`PUT <kibana host>:<port>/api/alerting/rule/<id>`
+
+`PUT <kibana host>:<port>/s/<space_id>/api/alerting/rule/<id>`
+
+[[rules-update-api-path-params]]
+==== Path parameters
+
+`id`::
+  (Required, string) The ID of the rule that you want to update.
+
+`space_id`::
+  (Optional, string) An identifier for the space. If `space_id` is not provided in the URL, the default space is used.
+
+[[rules-update-api-request-body]]
+==== Request body
+
+`name`::
+  (Required, string) A name to reference and search.
+
+`tags`::
+  (Optional, string array) A list of keywords to reference and search.
+
+`schedule`::
+  (Required, object) When to run this rule. Use one of the available schedule formats.
++
+._Schedule Formats_.
+[%collapsible%open]
+=====
+A schedule uses a key: value format.  {kib} currently supports the _Interval format_ , which specifies the interval in seconds, minutes, hours, or days at which to execute the rule.
+
+Example: `{ interval: "10s" }`, `{ interval: "5m" }`, `{ interval: "1h" }`, `{ interval: "1d" }`.
+
+=====
+
+`throttle`::
+  (Optional, string) How often this rule should fire the same actions. This will prevent the rule from sending out the same notification over and over. For example, if a rule with a `schedule` of 1 minute stays in a triggered state for 90 minutes, setting a `throttle` of `10m` or `1h` will prevent it from sending 90 notifications during this period.
+
+`notify_when`::
+  (Required, string) The condition for throttling the notification: `onActionGroupChange`, `onActiveAlert`,  or `onThrottleInterval`.
+
+`params`::
+  (Required, object) The parameters to pass to the rule type executor `params` value. This will also validate against the rule type params validator, if defined.
+
+`actions`::
+  (Optional, object array) An array of the following action objects.
++
+.Properties of the action objects:
+[%collapsible%open]
+=====
+  `group`:::
+    (Required, string) Grouping actions is recommended for escalations for different types of alerts. If you don't need this, set the value to `default`.
+
+  `id`:::
+    (Required, string) The ID of the action that saved object executes.
+
+  `connector_type_id`:::
```

Comment:
Remove `connector_type_id`

Reply:
Fixed in https://github.com/elastic/kibana/pull/94745/commits/ce20b691801950bb35c56bfc805432b6f23a1aa6.